h2o-persist-gcs/src/main/java/water/persist/StringCache.java

```diff
@@ -0,0 +1,33 @@
+package water.persist;
+
+import java.util.ArrayList;
+import java.util.List;
+
+abstract class StringCache {
+  private static final long TIMEOUT_MILLIS = 60 * 1000;
```

Comment:
I'm not sure if 60 seconds is a good or bad value. Should we discuss this before merging ?

Reply:
Initially, I was inspired by 5 mins in AWS connector, but IMHO it's a little bit too much, so I've reduced it to 1 min. I have no strong opinion on that, so feel free to change that.

Reply:
1 minute seems to work fine for local testing Not sure about larger buckets. Let's leave it to 1 minute.